### PR TITLE
Create pipeline monitoring docs

### PR DESCRIPTION
**Pull request by alexanderahn**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207679086948193

And update alternative schedulers formatting.

### CODE REVIEW

File: docs/Spark/pipeline-monitoring/use-pipeline-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+After your scheduled or ad-hoc run, you can view detected failures on your Pipeline. Gems with an error or warning are highlighted on the Visual canvas.
+
+- Hover over the Gem to see failure details, such as run duration and the exception.
+
+![Gem failture-details](img/gem-failure-details.png)
+
+- You can click **Open Logs** to open the Runtime Logs of that Gem.
+
+From the Runtime Logs panel, you can see other runs and jobs, including those that have succeeded. You can filer or sort the logs, and adjust the view.
```
[the remark this answers is not in the document]
Comment (alexanderahn) [reply docs/Spark/pipeline-monitoring/use-pipeline-monitoring.md:25]: Done.

File: docs/Spark/pipeline-monitoring/use-pipeline-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+- Click **More details** to see the entire exception details.
+
+![View runtime logs](img/view-runtime-logs.png)
+
+## Troubleshoot and fix problems
+
+From the Runtime Logs panel, you can swap the tabs by clicking **Problems**. The Problems tab shows you a list of all the Warnings or Errors, where they can be found, and brief descriptions of each.
```
[the remark this answers is not in the document]
Comment (alexanderahn) [reply docs/Spark/pipeline-monitoring/use-pipeline-monitoring.md:33]: Done.

File: docs/Spark/pipeline-monitoring/enable-pipeline-monitoring.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+tags:
+  - how-to
+  - setup
+  - monitoring
+  - spark
+---
+
+To use Pipeline Monitoring, you must enable it by updating the Prophecy Library version in your Project dependencies, and turn on the Pipeline Monitoring flag in Pipeline Settings.
+
+## Update Prophecy Library version
```
Comment (prongs) [inline docs/Spark/pipeline-monitoring/enable-pipeline-monitoring.md:15]: This only mentions python projects. For scala projects, we need to mention scala prophecy libs version. 

Let's mention 8.0.23 as the minimum version.

Comment (alexanderahn) [reply docs/Spark/pipeline-monitoring/enable-pipeline-monitoring.md:15]: Done.

**Review by poojaSprophecy — APPROVED** [review]

**Pull request merged**